packages/react-search-ui-views/src/__tests__/__snapshots__/Result.test.js.snap

```diff
@@ -1,5 +1,41 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`filters out arbitrary fields from results, and does not render them 1`] = `
```

Comment:
Just curious / for my own edification (I haven't used Jest's snapshots very much in the past, although now I'm thinking I should!) - why have the string within `exports['foo']` be a backtick/interpolated string? I get why the JSX itself is (for multiple lines), but less so the array key. Is it just a Jest standard?

Reply:
@constancecchen These are actually auto-generated by Jest, so not sure why they do that. Jest snapshots are great *sometimes*. There are mixed opinions, for sure.

Reply:
Ohhh gotcha gotcha, super interesting to know!